HQSmokeTests/settings-sample.cfg

```diff
@@ -6,3 +6,4 @@ bs_user =
 bs_key =
 staging_auth_key =
 prod_auth_key =
+password_mail_yahoo= "jakeisbest"
```

Comment:
The password should not be exposed! Also, were we not able to set yahoo's password the same as our webuser login?